libs/portals/admin/air-discount-scheme/src/screens/CreateDiscount/CreateDiscount.tsx

```diff
@@ -122,6 +146,7 @@ const AdminCreateDiscount = () => {
                 nationalId: nationalId.replace('-', ''),
                 postalcode: parseInt(postalcode, 10),
                 comment,
+                numberOfDaysUntilExpiration: parseInt(length, 10),
```

Comment:
Default value of `length` is "". What happens if you submit without interacting with the dropdown?